### ISSUE TITLE
Failed to mount component

### ISSUE THREAD

**Issue by sdonnet** [cscan/vue-excel-editor#91]
Hi,

I got a "[Vue warn]: Failed to mount component: template or render function not defined", when using/rendering VueExcelEditor.
I have no visual of the grid.

My template is :  

```
            <b-tab id="chapter2">            
               <template slot="title">
                  <big><i class="icon-list"></i> {{ $t('titles.prices') }}</big>
               </template>
               <template>
                  <vue-excel-editor ref="grid" v-model="jsonData">
                  </vue-excel-editor>
               </template>
            </b-tab>
```

And my script is :  

```
import VueExcelEditor from 'vue-excel-editor';
export default {
   name: 'CardUpdatableCustomerreseller',
   extends: CardUpdatable,
   components: {
      VueExcelEditor,
   },
   data: function () {
      return {
         jsonData: [
               ['', 'Tesla', 'Mercedes', 'Toyota', 'Volvo'],
               ['2019', 10, 11, 12, 13],
               ['2020', 20, 11, 14, 13],
               ['2021', 30, 15, 12, 13]
         ],
```

I don't see what I am missed.

Any help will be greatly appreciated !!!

Sylvain

**Comment by darksectordds** [comment]
I have some issue but on version 1.3.94 that why I started looking for a solution to the problem and went to this repository.
One of the ugly solutions if you connect it globally.

```
import VueExcelEditor from 'vue-excel-editor'
Vue.use(VueExcelEditor)
```

In my case, plugin-syntax-dynamic-import is used, which is dynamically in routes.js connects the required component that used local import VueExcelEditor which caused this error. I haven't found an elegant solution yet. While busy with other things to be honest... 

PS: sorry for my bad English.

**Comment by cscan** [comment]
No idea using in template

**Comment by darksectordds** [comment]
I solved my problem by bypassing _main.js_ and connecting the necessary components directly. 

```js
import VueExcelEditor from 'vue-excel-editor/src/VueExcelEditor';
import VueExcelColumn from 'vue-excel-editor/src/VueExcelColumn';
```

Now in my case, the error _"[Vue warn]: Failed to mount component: template or render function not defined"_ has disappeared and the **`vue-excel-editor'** component is connected dynamically.

I reduced the loaded scripts for those users who do not need it by almost 2.5 MB in dev version.
